source/ceylon/time/internal/GregorianDateTime.ceylon

```diff
@@ -343,17 +343,10 @@ shared class GregorianDateTime( date, time )
         return Instant( instant.millisecondsOfEpoch - timeZone.offset(instant) );
     }
 
-    "Returns _true_ if given value is same type, date and time."
+    "Two `DateTime`s are considered equals if they represent the same 
+     [[Date]] and [[Time]]."
     shared actual Boolean equals( Object other ) {
-        if (is GregorianDateTime other) {
-            if (this === other){
-                return true;
-            }
-
-            return date == other.date 
-                && time == other.time;
-        }
-        return false;
+        return (super of DateTime).equals(other);
```

Comment:
Same here. If all it does is invoke super.equals, there's no point overriding this method.